pkg/notifiers/jira/jira.go

```diff
@@ -64,17 +64,44 @@ type jira struct {
 	unknownMap map[string]interface{}
 }
 
+type issueTypeResult struct {
+	StartAt    int                     `json:"startAt"`
+	MaxResults int                     `json:"maxResults"`
+	Total      int                     `json:"total"`
+	IssueTypes []jiraLib.MetaIssueType `json:"values"`
+}
+
+func getIssueTypes(client *jiraLib.Client, project string) ([]jiraLib.MetaIssueType, error) {
+	urlPath := fmt.Sprintf("rest/api/2/issue/createmeta/%s/issuetypes", project)
```

Comment:
Would be nice to have a comment here explaining why we have a random URL instead of using the library methods. Maybe with a link to the deprecation notice: https://confluence.atlassian.com/jiracore/createmeta-rest-endpoint-to-be-removed-975040986.html